core/templates/pages/admin-page/roles-tab/admin-roles-tab.directive.ts

```diff
@@ -142,9 +142,9 @@ angular.module('oppia').directive('adminRolesTab', [
             formResponse.category, formResponse.username,
             formResponse.languageCode
           ).then(() => {
-            ctrl.setStatusMessage(
-              'Successfully added "' + formResponse.username + '" as ' +
-              formResponse.category + ' reviewer.');
+            ctrl.setStatusMessage(              
+              'Allowed "' + formResponse.username + '" to ' +
```

Comment:
Not sure how the log would look like for other options. Can you share the log for each option?